File: src/Samvaad/services/polyglot/hello.js
const args = process.argv.slice(2);

if (args.length === 0) {
    console.log(JSON.stringify({ error: "No arguments provided" }));
    process.exit(1);
}

try {
    const data = JSON.parse(args[0]);
    const name = data.name || 'Unknown';
    
    const response = {
        status: "success",
        lang: "Node.js",
        greeting: `Hello ${name} from Node.js!`,
        received_data: data
    };
    
    console.log(JSON.stringify(response));
} catch (e) {
    console.log(JSON.stringify({ error: e.message }));
}

File: src/Samvaad/services/polyglot/hello.py
import sys
import json

def main():
    if len(sys.argv) < 2:
        print(json.dumps({"error": "No arguments provided"}))
        return
        
    try:
        data = json.loads(sys.argv[1])
        name = data.get('name', 'Unknown')
        
        response = {
            "status": "success",
            "lang": "Python 3",
            "greeting": f"Hello {name} from Python!",
            "received_data": data
        }
        print(json.dumps(response))
    except Exception as e:
        print(json.dumps({"error": str(e)}))

if __name__ == "__main__":
    main()
